modules/ROOT/pages/asset-details.adoc

```diff
@@ -73,6 +74,37 @@ image::ex2-rest-ftns.png[Screenshot - REST API function buttons in left nav bar]
 This is similar to using a program such as Postman to specify custom HTTP headers and test each API's method using the API's endpoint.
 . To connect the API to one of your applications and use the API to send and receive data, click *Request access*.
 
+== View Conformance Status 
+
+You can view governance conformance status in Exchange for versions of API definitions. The status is one of the following:
+ 
+* A conformance badge is displayed for a selected version of an API.
+* In the *Manage Versions* page, a *Conformance* column shows conformance status for each version of the API. 
+* A *Conformance Status* page appears for all REST APIs and AsyncAPIs. If a selected version of an API has been validated against governance rulesets, the conformance status page gives conformance information and gives options to fix nonconformant versions.
+
+
+To view conformance status:
+
+. Select an API with governance rulesets.
```

Comment:
We should document how a user can find such an API (an API with governance rulesets) - if there is a way to discover those APIs

Reply:
In a later release, users will be able to search by conformance in Exchange.